scss/styles.scss

```diff
@@ -0,0 +1,95 @@
+$vars-nav-height: 90px;
+$vars-footer-height: 30vh;
+$vars-triangle-heigth: 90px;
+$vars-triangle-width: 100px;
+$vars-tech-logo-size: 100px;
+$vars-font-big: 24px;
+$vars-font-medium: 16px;
+$vars-font-small: 12px;
+
+* {
+  font-family: Roboto;
+}
+
+.front-container {
+  display: flex;
+  flex-direction: row;
+  margin: 10px;
+}
+
+.back-container {
+  display: flex;
+  flex-direction: row;
+  margin: 10px;
+}
+
+.main-title {
+  font-size: $vars-font-big;
+  position: relative;
+  margin: 100px 10px 100px 100px;
+}
+
+.main-title:before {
```

Comment:
use `::before` instead. although this works `:` is used for pseudo-classes and `::` is used for pseudo-elements. It works just because Chrome is cool enough to understand both syntaxes.

you can also take advantage of a property that sass has, instead of writing `.main-title` again you can just do this:
```
.main-title {
 main_title_properties
  ...
  ...

  &::before {
    main_title_before_properties
    ...
    ...
  }
}
```